docs/developer-guide/ide-setup/eclipse.md

```diff
@@ -70,4 +70,4 @@ Select `strongbox-masterbuild` project, Right click -> Maven -> Update Project
 [AQL groovy and twig Build Path]: {{assets}}/screenshots/ide/eclipse-setup/15-eclipse-aql-groovy-twig-to-build-path.png "AQL groovy and twig Build Path"
 [AQL groovy libraries in Build Path]: {{assets}}/screenshots/ide/eclipse-setup/16-eclipse-aql-groovy-libraries.png "AQL groovy libraries in Build Path"
 [AQL groovy project nature]: {{assets}}/screenshots/ide/eclipse-setup/17-eclipse-aql-groovy-nature.png "AQL groovy project nature"
-[masterbuild Maven update]: {{assets}}/screenshots/ide/eclipse-setup/18-eclipse-masterbuild-maven-update.png "masterbuild Maven update"
\ No newline at end of file
+[masterbuild Maven update]: {{assets}}/screenshots/ide/eclipse-setup/19-updated-eclipse-masterbuild-maven-update.png "masterbuild Maven update"
```

Comment:
Can't you just use the old link and just overwrite this file, or am I missing something...?

Reply:
Yep I can. I was reluctant to delete any existing asset. I can make that change certainly.